okrs/2019-q3.md

```diff
@@ -0,0 +1,27 @@
+## Objective: Projects that depend on IPLD are using new interfaces and specifications
+
+* Complete unixfsv2 implementation and relevant `iq` functionality (@mikeal)
+* Document `iq` and write a blog post (@mikeal)
+* Complete selector work necessary for GraphSync (@warpfork)
+* TODO: go-ipld-prime OKR's
+* TODO: schema OKR's
+
+## Objective: Building applications on IPLD is simple and fun
+
+* Build a delicio.us clone w/ IPLD (@mikeal)
+* Complete STAC geo data IPLD demo (@vmx)
```

Comment:
For the actual OKRs I'd like to make it more concrete. The goal is to have a browseable catalog like https://cbers.stac.cloud/ but with the data stored in IPLD. There's surely more things that could be done, so I might come back to this project later (e.g. if there's interest from the STAC people), but probably not in Q3.